UefiPayloadPkg/Library/PciSegmentInfoLibAcpiBoardInfo/PciSegmentInfoLibAcpiBoardInfo.c

```diff
@@ -182,6 +220,14 @@ Get_UPLSegInfo (
   //
   UplSegmentInfo = (UPL_PCI_SEGMENT_INFO_HOB *)GET_GUID_HOB_DATA (GuidHob);
   if (UplSegmentInfo->Count <= (GET_GUID_HOB_DATA_SIZE (GuidHob) - sizeof (UPL_PCI_SEGMENT_INFO_HOB)) / sizeof (UPL_SEGMENT_INFO)) {
+    GuidHob = GetFirstGuidHob (&gUefiAcpiBoardInfoGuid);

```

Comment:
FdtParserLib.c has updated UplSegmentInfo->SegmentInfo[*].BaseAddress basing on pci-rb->reg() property, may I know why we need to update again here by another HOB?

https://github.com/tianocore/edk2/blob/master/UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c#L761

Reply:
just in case , if the reg attribute wasn't exist.

Reply:
ok. since pci-rb->reg() is defined by spec as "required" property, we can remove such workaround.